MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -267,6 +267,11 @@
 				inlineUniformBlockFeatures->descriptorBindingInlineUniformBlockUpdateAfterBind = true;
 				break;
 			}
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_IMAGELESS_FRAMEBUFFER_FEATURES: {
+				auto* imagelessFramebufferFeatures = (VkPhysicalDeviceImagelessFramebufferFeaturesKHR*)next;
+				imagelessFramebufferFeatures->imagelessFramebuffer = true;
+				break;
+			}
```

Comment:
I typically don't add code for this if the feature enable isn't checked elsewhere.

Reply:
Oops, wrong spot. This is the place you absolutely need to add this, because this is where we report the supported features of the device! Sorry about that. I meant to attach this comment to the other patch hunks.